mule-apikit-tools/mule-apikit-scaffolder/src/main/java/org/mule/tools/apikit/output/GenerationModel.java

```diff
@@ -203,4 +202,13 @@ public String getFlowName() {
   public int compareTo(@Nonnull GenerationModel generationModel) {
     return this.getName().compareTo(generationModel.getName());
   }
+
+  public List<String> getUriParameters() {
+    List<String> result = new ArrayList<>();
+    for (String part : Lists.newArrayList(resource.getResolvedUri(version).split("/"))) {
+      if (part.startsWith("{") && part.endsWith("}"))
+        result.add(part.substring(1, part.length() - 1));
+    }
+    return result;
+  }
```

Comment:
IResource has a method called `getResolvedUriParameters()`, could it be used here?

Reply:
getResolvedUriParameters() return an empty map, although the resource has uri params